apps/server/src/modules/tool/service/validation/external-tool-validation.service.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('ExternalToolValidation', () => {
 		});
 
 		describe('when external tool config has oauth config', () => {
-			it('should not find a tool with this client id', async () => {
+			it('should not find a tool with this unique client id', async () => {
```

Comment:
Please use the setup function structure for all 3 touched it`s.

Reply:
added setup() and used it on all tests in that file